### PR TITLE
Create CA Assemblymember Yamls

### PR DESCRIPTION
**Pull request by OdeclasV**

### CODE REVIEW

File: members/SCAL1756d0e0.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+          value: $MESSAGE
+          options:
+            max_length: 2000
+          required: true
+    - select:
+        - name: Topic
+          selector: select#IssueListBox
+          value: $TOPIC
+          options:
+            -- select issue --: -- select issue --
```
Comment (tomdionesotes) [inline members/SCAL1756d0e0.yaml]: shouldn't need this line, it's not a valid choice

File: members/SCAL83007e2a.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
+bioguide: SCAL83007e2a
+contact_form:
+  method: post
+  action: ""
+  steps:
+    - visit: "https://lcmspubcontact.lc.ca.gov/PublicLCMS/ContactPopup.php?district=AD45"
+    - fill_in:
+        - name: Firs tName
```
Comment (tomdionesotes) [inline members/SCAL83007e2a.yaml]: le typo

File: members/SCALc70d3e9b.yaml [hunk 3]
```diff
@@ -0,0 +1,64 @@
+bioguide: SCALc70d3e9b
+contact_form:
+  method: post
+  action: ""
+  steps:
+    - visit: "https://lcmspubcontact.lc.ca.gov/PublicLCMS/ContactPopup.php?district=AD54"
+    - fill_in:
+        - name: Firs tName
```
Comment (tomdionesotes) [inline members/SCALc70d3e9b.yaml]: typoo

File: members/SCAL69f747c5.yaml [hunk 4]
```diff
@@ -0,0 +1,64 @@
+bioguide: SCAL69f747c5
+contact_form:
+  method: post
+  action: ""
+  steps:
+    - visit: "https://lcmspubcontact.lc.ca.gov/PublicLCMS/ContactPopup.php?district=AD39"
+    - fill_in:
+        - name: Firs tName
```
Comment (tomdionesotes) [inline members/SCAL69f747c5.yaml]: tyypo

File: members/SCAL1756d0e0.yaml [hunk 5]
```diff
@@ -0,0 +1,64 @@
+bioguide: SCAL1756d0e0
+contact_form:
+  method: post
+  action: ""
+  steps:
+    - visit: "https://lcmspubcontact.lc.ca.gov/PublicLCMS/ContactPopup.php?district=AD51"
+    - fill_in:
+        - name: Firs tName
```
Comment (tomdionesotes) [inline members/SCAL1756d0e0.yaml]: typpo

**Comment by OdeclasV** [comment]
Sure thing, I'm on it @tomdionesotes

**Comment by OdeclasV** [comment]
These are some of the most commons characters being blacklisted that I found. Let me know if there are other I should add @tomdionesotes

**Pull request merged**